src/matlab2tikz.m

```diff
@@ -1502,6 +1502,16 @@ case guitypes()
 
     options = setAxisTicks(m2t, options, axis, pgfTicks, pgfTickLabels, ...
         hasMinorTicks, tickDirection, isDatetimeTicks);
+        
+    options = setAxisTickLabelStyle(m2t, options,axis,handle);
+end
+
+function options = setAxisTickLabelStyle(m2t, options,axis,handle)
+ keywordTickLabelRotation = [upper(axis), 'TickLabelRotation'];
```

Comment:
This only exists in MATLAB R2014b and newer. In Octave and R2014a or older, this throws errors like `The name 'XTickLabelRotation' is not an accessible property for an instance of class 'axes'.`.

To avoid this, you can handle this with our `getAndCheckDefault` or `getOrDefault` functions:
```matlab
key = [upper(axis), 'TickLabelRotation'];
% either:
[rotation, isDefaultRotation] = getAndCheckDefault('axes', handle, key, 0);
if ~isDefaultRotation
    ...
end
% or:
rotation = getOrDefault(handle, key, 0);
if rotation ~= 0
    ...
end
```